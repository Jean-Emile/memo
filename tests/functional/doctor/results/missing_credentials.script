{
    "leftovers": [],
    "networks": [
        {
            "linked": true,
            "name": "bob/aws_network",
            "sane": false,
            "silos": [
                {
                    "name": "aws_storage",
                    "reason": "credentials are missing",
                    "sane": false,
                    "type": "S3",
                    "warning": false
                }
            ],
            "warning": false
        },
        {
            "linked": true,
            "name": "bob/fs_network",
            "sane": true,
            "warning": false
        }
    ],
    "sane": false,
    "silos": [
        {
            "name": "aws_storage",
            "reason": "credentials are missing",
            "sane": false,
            "type": "S3",
            "warning": false
        },
        {
            "name": "fs_storage",
            "reason": "\"/tmp/\" is readable and writable",
            "sane": true,
            "type": "filesystem",
            "warning": false
        }
    ]
}
