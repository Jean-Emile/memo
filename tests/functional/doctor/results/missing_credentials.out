[OK] User
[ERROR] Storage resources:
  [ERROR] aws_storage is faulty because credentials are missing
[ERROR] Networks:
  [ERROR] bob/aws_network is faulty because "aws_storage" (credentials are missing) storage resource is faulty
[ERROR] Volumes:
  [ERROR] bob/aws_volume is faulty because network "bob/aws_network" is faulty
[ERROR] Drives:
  [ERROR] bob/aws_drive is faulty because volume "bob/aws_volume" is faulty
[OK] Leftovers
